src/base/QXmppMixIq.h

```diff
@@ -20,7 +21,9 @@ class QXMPP_EXPORT QXmppMixIq : public QXmppIq
         ClientLeave,
         Join,
         Leave,
-        UpdateSubscription,
+#if QXMPP_DEPRECATED_SINCE(1, 7)
+        UpdateSubscription [[deprecated("Use QXmppMixSubscriptionUpdate")]],
+#endif
         SetNick,
```

Comment:
This causes `SetNick` to have a different value when depreacated symbols are disabled. Assign a fixed value to `SetNick`.